packages/perspective-viewer-d3fc/src/js/charts/ohlcCandle.js

```diff
@@ -16,22 +16,36 @@ import withGridLines from "../gridlines/gridlines";
 import {hardLimitZeroPadding} from "../d3fc/padding/hardLimitZero";
 import zoomableChart from "../zoom/zoomableChart";
 import nearbyTip from "../tooltip/nearbyTip";
-import {seriesUpColors, seriesDownColors} from "../series/seriesColors";
+import {colorScale, setOpacity} from "../series/seriesColors";
 import {colorLegend} from "../legend/legend";
 
 const isUp = d => d.closeValue >= d.openValue;
 
 function ohlcCandle(seriesCanvas) {
     return function(container, settings) {
-        const data = ohlcData(settings, filterDataByGroup(settings));
+        const srcData = ohlcData(settings, filterDataByGroup(settings));
 
-        const keys = data
+        const bollinger = fc.indicatorBollingerBands().value(d => d.openValue);
```

Comment:
Are there a lot of `bollinger` specific calculations we do even if we draw the ohlc or candle stick charts? I guess this only happens at the beginning.